tests/conftest.py

```diff
@@ -82,7 +82,7 @@ def _requests_post(url, **kwargs):
 def select_first(dash_duo, selector):
     parameter_selector_input = dash_duo.find_element(selector)
     options = parameter_selector_input.text.split("\n")
-    dash_duo.click_at_coord_fractions(parameter_selector_input, 0.1, 0.1)
+    dash_duo.click_at_coord_fractions(parameter_selector_input, 0.1, 0.05)
```

Comment:
Had to change this because when using this function to select the first element from the parameter selection list it would actually select the second one